### ISSUE TITLE
linkage map for each parent

### ISSUE THREAD

**Issue by jtahir** [cran/polymapR#1]
Hi
Can polymapR generate separate linkage maps for each parent instead of integrated/consensus...
thanks

**Comment by YanlinLiao** [comment]
Yes. In the combine linkage_list step, you do not combine them. Keep two parents' linkage_list separate and then call MDSmap for ordering

**Comment by gaborcsardi** [comment]
Hi, this is a read only mirror of CRAN, please see the package authors and potentially URLs in the `DESCRIPTION` file. Thanks!
